jupytext/jupytext_config.py

```diff
@@ -0,0 +1,64 @@
+from argparse import ArgumentParser
+
+from .labconfig import LabConfig
+
+# list of subcommands - filled by SubCommand.__init__
+SUBCOMMANDS = []
+
+
+class SubCommand:
+    def __init__(self, name, help):
+        self.name = name
+        self.help = help
+        SUBCOMMANDS.append(self)
+
+    def main(self, args):
+        """
+        return 0 if all goes well
+        """
+        print(
+            f"{self.__class__.__name__}: redefine main() to implement this subcommand"
+        )
+        return 1
+
+
+class List(SubCommand):
+    def __init__(self):
+        super().__init__("list", "Display current settings in labconfig/")
```

Comment:
Maybe `list-default-viewer` or `show-default-viewer` would be a more precise name? Or else do you want to show the full jupyterlab configuration? Do you want to show it formatted in JSON or in a more user friendly format?

Reply:
see ed8055f547125cf408b468d3e9e012b78f17dcf8